tests/stencils/test_fields.py

```diff
@@ -58,11 +58,8 @@ def h_offsets(
 ):
     with levels_upward:
         with sparse[Edge > Cell > Edge]:
-            a = b[Edge > Cell > Edge] + c  # no offsets, defaults to Edge > Cell > Edge
-            a = (
-                b[Edge > Cell > Edge] + c[Edge > Cell > Edge]
-            )  # verbose version of the above
-            a = b[Edge] + c[Edge > Cell > Edge]  # center access for dense field
+            a = b[Edge > Cell > Edge] + c  # no offsets, defaults to Edge > Cell > Edge            
+            a = b[Edge]                    # center access for dense field
```

Comment:
```suggestion
            a = b[Edge > Cell > Edge] + c
            a = b[Edge] + c
```
I think the comments are outdated and the original test always had `a = b + c`.